File: Code coach/Easy/Cheer creator/cheer_creator.cpp
#include <iostream>

int main()
{
	int yards;
	std::cin >> yards;

	if (yards > 10)
	{
		std::cout << "High Five";
	}
	else if (yards < 1)
	{
		std::cout << "shh";
	}
	else
	{
		for(int i = 0; i < yards; i++)
		{
			std::cout << "Ra!";
		}
	}

	return 0;
}

File: Code coach/Easy/Cheer creator/CheerCreator.swift
if let input = readLine(), let yards = Int(input) {
	if yards > 10 {
		print("High Five")
	} else if yards < 1 {
		print("shh")
	} else {
		for _ in 1...yards {
			print("Ra!", terminator: "")
		}
	}
}